RoyaltiesProcessorDelivery/src/index.ts

```diff
@@ -492,6 +520,13 @@ yargs(hideBin(process.argv))
         throw 'invalid configuration'
       }
 
+      if (useReservoirApi && !flagship && !contract) {
+        console.error(
+          '[ERROR] Reservoir API mode currently only supports --flagship or --contract mode'
+        )
+        throw 'invalid configuration'
+      }
```

Comment:
is this an actual constraint with the reservoir API, or just our logic internally?

Reply:
internal logic - was gonna tackle untangling that in a different PR

Reply:
👍